<?xml version="1.0"?>
<launch>
    <node pkg="mir_compute_transform" type="compute_transform_node"
          name="compute_transform" output="screen">

        <!-- the cycle rate of the node (in Hz) -->
        <param name="loop_rate" type="double" value="50"/>

        <param name="world_frame_rotation" type="double" value="-0.707"/>

        <param name="wait_for_transform" type="double" value="0.1"/>

        <param name="target_frame" type="str" value="arm_link_0"/>

        <!---param name="reference_frame" type="str" value="gripper_tip_link"/-->
        <param name="reference_frame" type="str" value="arm_link_5"/>

        <param name="task_frame" type="str" value="task_link"/>
    </node>
</launch>
